proto/viam/app/data/v1/data.proto

```diff
@@ -177,10 +177,18 @@ message BinaryDataByFilterResponse {
   string last = 3;
 }
 
-// BinaryDataByFilterRequest requests the data and metadata of binary (image + file) data by file ids
+message BinaryID {
+  string file_id = 1;
+  string organization_id = 2;
+  string location_id = 3;
+}
+
+// BinaryDataByFilterRequest requests the data and metadata of binary (image + file) data by binary ids
 message BinaryDataByIDsRequest {
-  repeated string file_ids = 1;
-  bool include_binary = 2; // by default
+  bool include_binary = 2;
+  repeated BinaryID binary_ids = 3;
+
+  repeated string file_ids = 1 [deprecated = true]; // Replaced by binary_ids
```

Comment:
These will be removed and reserved in a later PR that removes all deprecated Data fields from the entire scoped cleanup.